api/actions/read_app_logs.go

```diff
@@ -35,21 +36,23 @@ func (a *ReadAppLogs) Invoke(ctx context.Context, authInfo authorization.Info, a
 		defaultLogLimit = 100
 	)
 
+	logger := apis.AddCorrelationIDToLogger(ctx, a.logger)
```

Comment:
All loggers have to be initialized in the new way now, which is repetitive and can be forgotten. Could we create the logger in the auth-aware handler wrapper and pass it to the handler explicitly as an argument?

Maybe it would make sense to propagate the logger with the correlation id from the handler to all places that do logging, e.g. the pod repository, the retrying k8s client, etc. What do you think?

Reply:
We could also merge this as is and have a chore to do what we suggested above. Let's discuss at conv street tomorrow.